summingbird-storm/src/main/scala/com/twitter/summingbird/storm/collector/AggregatorOutputCollector.scala

```diff
@@ -46,27 +45,33 @@ class AggregatorOutputCollector[K, V: Semigroup](
     flushExecTimeCounter.incrBy(Time.now.inMillis - startTime.inMillis)
   }
 
-  private def convertToSummerInputFormat(flushedCache: CMap[_, _]): CMap[Int, CMap[_, _]] =
-    flushedCache.groupBy { case (k, _) => summerShards.summerIdFor(k) }
+  private def convertToSummerInputFormat(flushedCache: Map[K, (Seq[Object], V)]): TraversableOnce[OutputTuple] =
+    flushedCache.groupBy {
+      case (k, _) => summerShards.summerIdFor(k)
+    }.map {
+      case (index: AggKey, m: Map[K, (OutputMessageId, V)]) =>
```

Comment:
`case f: Foo` is a pattern match on type. Can we remove the types to be clear we are not doing that? If you want to annotate, you can do in the next line, or do:
```scala
.map { tup: (AggKey, Map[...]) =>
  val (index, m) = tup
```